worker/tox.ini

```diff
@@ -4,7 +4,8 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py24, py25, py26, py27
+envlist = py24, py25, py26, py27, py35, py36, py37
```

Comment:
Pretty sure tox isn't used and that's why this hasn't been updated.

Reply:
Yes, but I also didn't figure out how else the worker is being tested for py2.x regressions